src/test/resources/hudson/tasks/junit/JENKINS-41134/TestSuite_second.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+    <testsuite name="TestSuite_second" tests="1" errors="0" failures="1" skip="0" timestamp="2012-04-05T10:50:00">

```

Comment:
my gut says that the probability is low (even though the the timestamp has only second granularity).
At least for the reported failure (as provisioning the windows node takes a while).

Now for something running tests in a parallel branch on the same node (e.g. with different browsers or some flag) then the probability is probably higher - but I think that can be handled as a different issue - as it may break use cases where a user runs unit tests, and then integration (maven failsafe) on the same workspace and runs the junit step in between...